src/main/java/frc/robot/Robot.java

```diff
@@ -144,8 +145,10 @@ public void disabledExit() {
   @Override
   public void autonomousInit() {
     Shuffleboard.startRecording();
-    if (SubsystemConstants.DRIVEBASE_ENABLED && AutoLogic.getSelectedAuto() != null) {
-      AutoLogic.getSelectedAuto().schedule();
+    if (SubsystemConstants.DRIVEBASE_ENABLED) {
+      Commands.waitSeconds(AutoLogic.autoDelayEntry.get().getDouble())
```

Comment:
While you're here, it'd be nice to skip the waitSeconds call if the autoDelayEntry entry is 0. As written (and as currently implemented), I think this delays the auto routine by a code cycle since the WaitCommand has to run for a single cycle to detect that it has nothing to do.